bpf/lib/nodeport.h

```diff
@@ -838,8 +838,8 @@ int tail_nat_ipv64(struct __ctx_buff *ctx)
 #endif /* ENABLE_NAT_46X64_GATEWAY */
 
 static __always_inline int
-nodeport_rev_dnat_ingress_ipv6(struct __ctx_buff *ctx, struct trace_ctx *trace,
-			       __s8 *ext_err)
+nodeport_rev_dnat_ipv6(struct __ctx_buff *ctx, struct trace_ctx *trace,
```

Comment:
Could you please also update the naming for the corresponding IPv4 functions? They're used in the same way, and I think we want consistent naming (otherwise it gets even more confusing).